common/models.json

```
@@ -0,0 +1,17 @@
+{
+    "T1B1": {
+        "name": "Model One",
+        "colors": {}
+    },
+    "T2T1": {
+        "name": "Model T",
+        "colors": {}
+    },
+    "T2B1": {
+        "name": "Model R",
+        "colors": {
+            "0": "Black",
+            "1": "White"
```

we would prefer lowercase, because we will primary use it for filenames
```suggestion
            "0": "black",
            "1": "white"
```

---

I take it back, we can also use a number as a key